### ISSUE TITLE
[BUG]500 error occurs when querying logs in ui

### ISSUE THREAD

**Issue by weizhuangzhi** [bunkerity/bunkerweb#613]
**Description**
When querying the log in the UI, the browser will report a 500 error.

**How to reproduce**
On the Logs page, select From date and click submit date, it will appear.
The browser console will display: logs.js:333 GET http://******/manage/logs/local?from_date=1693756800000 500 (Internal Server Error)

**Logs**
```
[2023-09-05 15:33:12 +0800] [14826] [❌] Error handling request /manage/logs/local?from_date=1693756800000
Traceback (most recent call last):
  File "/usr/share/bunkerweb/deps/python/gunicorn/workers/base_async.py", line 55, in handle
    self.handle_request(listener_name, req, client, addr)
  File "/usr/share/bunkerweb/deps/python/gunicorn/workers/ggevent.py", line 128, in handle_request
    super().handle_request(listener_name, req, sock, addr)
  File "/usr/share/bunkerweb/deps/python/gunicorn/workers/base_async.py", line 108, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/bunkerweb/deps/python/flask/app.py", line 2213, in __call__
    return self.wsgi_app(environ, start_response)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/bunkerweb/ui/src/ReverseProxied.py", line 68, in __call__
    return self.app(environ, start_response)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/bunkerweb/deps/python/flask/app.py", line 2193, in wsgi_app
    response = self.handle_exception(e)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/bunkerweb/deps/python/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/bunkerweb/deps/python/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/bunkerweb/deps/python/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/bunkerweb/deps/python/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/bunkerweb/deps/python/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/bunkerweb/ui/main.py", line 1270, in logs_linux
    logs_access = [
                  ^
  File "/usr/share/bunkerweb/ui/main.py", line 1271, in <listcomp>
    f"{datetime.strptime(line[line.find('[') + 1: line.find(']')], '%d/%b/%Y:%H:%M:%S %z').timestamp()} {line}"
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'notice' does not match format '%d/%b/%Y:%H:%M:%S %z'
```

What is the cause? Or did I configure it wrong somewhere?

**Comment by TheophileDiot** [comment]
Hi, thank you for opening this issue, I'll check what's wrong.

**Comment by fl0ppy-d1sk** [comment]
Hello @weizhuangzhi,

It should be fixed in the last [1.5.2 version](https://github.com/bunkerity/bunkerweb/releases/tag/v1.5.2). Feel free to open a new issue if needed.
